### ISSUE TITLE
A11y: No-JS Support for AV

### ISSUE THREAD

**Issue by 12** [bbc/simorgh#5066]
**Is your feature request related to a problem? Please describe.**
Following on from the merge of #4932, we need to schedule a developer accessibility swarm to validate the accessibility of the introduction of support for no JavaScript. The original PR was merged prior to a dev swarm to prevent it getting lost and too outdated, as we are waiting on a third-party dependency to fix hardware in order to complete the dev swarm.

**Describe the solution you'd like**
Perform an accessibility swarm on the media player component with JavaScript disabled.

**Comment by HarveyPeachey** [comment]
Some checks have been carried out already:
**Dev/A11y Swarm**

-https://bbc.github.io/accessibility-news-and-you/accessibility-news-and-testers
  - [ ]   ~Axe~ - Can't run Axe when JS is disabled
  - [x] Design
  - [x] Colour Contrast
  - [x] Colour blindness
  - [ ] Windows HCM 
  - [x] FF Colour Preferences
  - [x] Text 200%, Page 200%
  - [x] Icons

_Screen Readers (IE11) -
- [ ] ZoomText Magnifier/Reader
- [ ] Dragon NaturallySpeaking
- [ ] JAWS
- [ ] Read&Write
- [ ] NVDA

_Mac, iOS, and Android:_ 
- [ ] VoiceOver iOS (iPhone/iPad)
- [x] VoiceOver OS (Mac)
- [ ] TalkBack (Chrome on Android)

**Comment by HarveyPeachey** [comment]
This a11y swarm cannot be completed until IT Support and Atos have allowed given us the correct permissions to disable JS in IE11 for our a11y testing laptops.

**Comment by HarveyPeachey** [comment]
Unblocked, a11y laptop 176 now has permission to disable JS on IE11
